src/components/AttachmentModal.js

```diff
@@ -251,7 +251,7 @@ class AttachmentModal extends PureComponent {
                         shouldShowBorderBottom
                         shouldShowDownloadButton={this.props.allowDownload}
                         onDownloadButtonPress={() => {
```

Comment:
Let's move this to a method. Also we should add a comment to explain why we need `Keyboard.dismiss()`

Reply:
done, let me know if its okay